nilearn/datasets/func.py

```diff
@@ -1301,98 +1520,203 @@ def fetch_surf_nki_enhanced(n_subjects=10, data_dir=None,
 
     """
     if url is None:
-        url = 'https://www.nitrc.org/frs/download.php/'
+        url = "https://www.nitrc.org/frs/download.php/"
 
     # Preliminary checks and declarations
-    dataset_name = 'nki_enhanced_surface'
-    data_dir = _get_dataset_dir(dataset_name, data_dir=data_dir,
-                                verbose=verbose)
-    ids = ['A00028185', 'A00033747', 'A00035072', 'A00035827', 'A00035840',
-           'A00037112', 'A00037511', 'A00038998', 'A00039391', 'A00039431',
-           'A00039488', 'A00040524', 'A00040623', 'A00040944', 'A00043299',
-           'A00043520', 'A00043677', 'A00043722', 'A00045589', 'A00050998',
-           'A00051063', 'A00051064', 'A00051456', 'A00051457', 'A00051477',
-           'A00051513', 'A00051514', 'A00051517', 'A00051528', 'A00051529',
-           'A00051539', 'A00051604', 'A00051638', 'A00051658', 'A00051676',
-           'A00051678', 'A00051679', 'A00051726', 'A00051774', 'A00051796',
-           'A00051835', 'A00051882', 'A00051925', 'A00051927', 'A00052070',
-           'A00052117', 'A00052118', 'A00052126', 'A00052180', 'A00052197',
-           'A00052214', 'A00052234', 'A00052307', 'A00052319', 'A00052499',
-           'A00052502', 'A00052577', 'A00052612', 'A00052639', 'A00053202',
-           'A00053369', 'A00053456', 'A00053474', 'A00053546', 'A00053576',
-           'A00053577', 'A00053578', 'A00053625', 'A00053626', 'A00053627',
-           'A00053874', 'A00053901', 'A00053927', 'A00053949', 'A00054038',
-           'A00054153', 'A00054173', 'A00054358', 'A00054482', 'A00054532',
-           'A00054533', 'A00054534', 'A00054621', 'A00054895', 'A00054897',
-           'A00054913', 'A00054929', 'A00055061', 'A00055215', 'A00055352',
-           'A00055353', 'A00055542', 'A00055738', 'A00055763', 'A00055806',
-           'A00056097', 'A00056098', 'A00056164', 'A00056372', 'A00056452',
-           'A00056489', 'A00056949']
+    dataset_name = "nki_enhanced_surface"
+    data_dir = _get_dataset_dir(
+        dataset_name, data_dir=data_dir, verbose=verbose
+    )
+    ids = [
```

Comment:
This isn't related to reformatting but I'm wondering whether we should have helper functions whose only responsibility is to return the list of ids for a given dataset fetcher.

So basically something like this:

```python
# In the fetcher
ids = _get_nki_ids()

# Helper function
def _get_nki_ids():
    return [....]
```

 It's not super important, but these massive lists are breaking the function logic and force the reader to scroll up and down which can be annoying...
WDYT ?

Reply:
yes after the reformatting we could extract those long lists of IDs in a separate function.